packages/tradershub/src/features/cfd/constants.tsx

```diff
@@ -57,11 +57,11 @@ export const CFDPlatforms = {
     MT5: 'mt5',
 } as const;
 
-export const MarketType = {
-    ALL: 'all',
-    FINANCIAL: 'financial',
-    SYNTHETIC: 'synthetic',
-} as const;
+export enum MarketType {
```

Comment:
Question
Why ae you changing it to enum
What about others?
Have you changed others in different PR?